auv_sim/auv_sim_bringup/launch/inc/start_pool_world.launch.xml

```diff
@@ -16,8 +14,4 @@
     <arg name="verbose" value="true" />
   </include>
 
-  <node name="publish_world_models" pkg="uuv_assistants" type="publish_world_models.py"
-    output="screen">
-    <rosparam command="load" file="$(find auv_sim_bringup)/config/world_models.yaml" />
-  </node>
-</launch>
```

Comment:
Separated the launch files for each world to make it easier to add world-specific nodes